### ISSUE TITLE
Updates for AFNetworking 3

### ISSUE THREAD

**Issue by billburgess** [AFNetworking/AFAmazonS3Manager#105]
Since AFNetworking is the owner of this add-on, would it be possible to update this useful library to work on top of AFNetworking 3? Unfortunately it will require quite a bit more work than I'm comfortable doing for a pull request.

**Comment by akac** [comment]
Agreed. I'd love to get an AFAmazonS3Manager redesigned for AFNetworking 3.

**Comment by Seenu16** [comment]
+1

**Comment by walteryaron** [comment]
+1

**Comment by punithbm** [comment]
anyone found answer for this?

**Comment by sherwoodyao** [comment]
+1

**Comment by cwveysey** [comment]
+1

**Comment by ljfantin** [comment]
+1

**Comment by ljfantin** [comment]
anyone found answer for this?

**Comment by levey** [comment]
Creaeted PR In https://github.com/AFNetworking/AFAmazonS3Manager/pull/113
